### PR TITLE
Addition and small fix

### PR DESCRIPTION
**Pull request by eevan78**
Hi Miljan!
I have found some errors in links on README.md and I have contributed two more commands to git!

Great project!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ In the best case scenario **Version** column should contain the software version
...
 
 ## Contributors ✨
 
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification and is brought to you by these awesome [contributors](https://github.com/miljanuscumlic/terminal-commands/blob/git/CONTRIBUTORS.md).
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification and is brought to you by these awesome [contributors](https://github.com/miljanuscumlic/terminal-commands/blob/develop/CONTRIBUTORS.md).
 
 ## License
 
-This project is licensed under the Unlicense - see the [UNLICENSE](https://github.com/miljanuscumlic/terminal-commands/blob/git/UNLICENSE) file for details.
+This project is licensed under the Unlicense - see the [UNLICENSE](https://github.com/miljanuscumlic/terminal-commands/blob/develop/UNLICENSE) file for details.
```
Comment (miljanuscumlic) [inline README.md]: We will merge develop branch to a master, and then these links will not be correct again. Please change `develop` with `master` in those links.
We will keep in mind their temporary non-correctness until the branch gets merged into the master.

**Review by miljanuscumlic — CHANGES_REQUESTED** [review]
Hi Ivan,
Thanks for your contribution. This is looking good, but there is one small issue that should be addressed before I merge it. I left a comment.

File: README.md (hunk 1 above)
Comment (eevan78) [reply README.md]: That's a valid point. I missed it :) I will change branch to `master`.

**Review by miljanuscumlic — APPROVED** [review]
Great!

**Pull request merged**